pkg/mesos/taskio.go

```diff
@@ -287,9 +287,14 @@ func (t *TaskIO) attachContainerInput() error {
 			}()
 
 			for {
-				// TODO(bamarni): investigate on why "test_task:test_attach" fails if the buffer size below
-				// is greater than 1, this might be an issue with the term package or the way we use it.
-				buf := make([]byte, 1)
+				bufLen := 512
```

Comment:
Why 512 in particular?

Reply:
It's similar to https://github.com/mesos/mesos-go/blob/3c2ae7f09d530e0ecd62acd2c31a4d73ed9bdf55/api/v1/cmd/msh/app/app.go#L517

For the record the Python CLI had 1024: https://github.com/dcos/dcos-core-cli/blob/1c5f9faffe757abef90ec337fdacaf7b6c01c8f4/python/lib/dcos/dcos/mesos.py#L1644

Do you have any suggestions regarding this?

Reply:
Using the same file input, it took 22, then 23 seconds with the value bumped to 1024. Maybe there is a slight improvement if we bump it, though it also increases memory usage. I think the main improvement is not to leave it to something as low as 1.